README.md

```diff
@@ -129,6 +129,7 @@ You will probably need to run this for the `classes`, `subjects` and `reviews` c
 - **Samuel Yeboah** - Developer
 - **Helen Lei** - Developer
 - **Andrea Thia** - Designer
+- **Andrew Qian** - Developer
```

Comment:
Actually, could you re-order it and do half of the other newbies 
- tbd -> Megan Yap for apm 
- Michelle li -> julie jeong as Dev Advisor 
- move designers to be above dev advisor 
- add 1 new designer - Valerie Wong 
- remove Sameul 
- put Harry all the way down and rename him as "Coach" (for fun lol)

and then I'll have Jacqueline do the rest of the peeps 


TY!